tokens/tokens-eth.json

```diff
@@ -13791,6 +13791,26 @@
 			"twitter": "https://twitter.com/_Peculium",
 			"youtube": "https://www.youtube.com/watch?v=kg2vYFJ_R50"
 		}
+	},
+		{
+		"symbol": "PDATA",
+		"type": "ERC20",
+		"address": "0x0db03B6CDe0B2d427C64a04FeAfd825938368f1F",
+		"decimals": 18,
+		"name": "PDATA",
+		"ens_address": "",
+		"website": "https://opiria.io",
+		"logo": {
+			"src": "https://www.opiria.io/static/images/logo-solo.png",
+			"width": 95,
+			"height": 110,
+			"ipfs_hash": ""
+		},
+		"support": { "email": "sales@opiria.com", "url": "www.opiria.io" },
+		"social": {
+		
+			"telegram": "https://t.me/PDATAtoken"
```

Comment:
Please keep the format the same as the other tokens, if you don't have the links to the social medias, just leave them blank